pkg/varlinkapi/util.go

```diff
@@ -195,3 +196,13 @@ func makePsOpts(inOpts iopodman.PsOpts) shared.PsOptions {
 		Sync:      derefBool(inOpts.Sync),
 	}
 }
+
+func (v *VarlinkCall) RequiresUpgrade() (ok bool, err error) {
```

Comment:
Why bool, error? This can probably get away with just an error - a nil/not nil check replaces the bool

Reply:
@mheon You are correct but (bool, error) is more expressive given I'm doing a test rather than a transform.  I could change the signature to (bool) if it offends.